.github/workflows/audits.yml

```diff
@@ -63,10 +63,11 @@ jobs:
     strategy:
       matrix:
         workspace:
+          - deno
           - hotchocolate
-          - postgraphile
+          - lighthouse
           - pioneer
-          - deno
```

Comment:
Why reorder?

Reply:
I could not figure out where to put `lighthouse`, so I ordered the list alphabetically.

Reply:
On the bottom, lets make the changeset minimal.